app/operations/story_operations.rb

```diff
@@ -52,11 +42,9 @@ def after_save
     private
 
     def ensure_valid_state
-      if should_be_unstarted? estimate: params['estimate'], state: params['state']
-        params['state'] = 'unstarted'
-      elsif should_be_unscheduled? estimate: params['estimate'], type: params['story_type']
-        params['state'] = 'unscheduled'
-      end
+      return unless should_be_unscheduled? estimate: params['estimate'], type: params['story_type']
```

Comment:
nitpicking but, I think you can make the assignment directly in here like so:

```
params['state'] = 'unscheduled' unless should_be_unscheduled? estimate: params['estimate'], type: params['story_type']
```

Reply:
But in this way, the line will be very big and the `code climate` will break :cry:

Reply:
Hmm I was wondering if this would've happen.

Leave the way it is then, the code is pretty simple and easy to understand anyway 👍